src/components/FormField/FormField.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './FormField.module.scss';
+
+const FormField = ({
+  label,
+  type,
+  isEditable,
+  onChange,
+  value,
+  placeholder,
+}) => (
+  <div className={styles.FormField}>
+    <label>{label}</label>
+
+    {type === 'textarea' ? (
+      <textarea
+        readOnly={!isEditable}
+        onChange={isEditable ? onChange : null}
+        placeholder={placeholder}
+        value={value}
```

Comment:
maybe it's better to use object to keep this:
```js
const inputProps = {
  readOnly: !isEditable,
  onChange: isEditable ? onChange : null,
  placeholder,
  value
}

<textarea {...inputProps}/>
```

Reply:
Done. But had to switch off `react/jsx-props-no-spreading` linting rule